enterprise/frontend/src/embedding-sdk/lib/theme/embedding-color-palette.ts

```diff
@@ -45,19 +58,20 @@ const originalColors = { ...colors };
  * @param appPalette color palette from the admin appearance settings
  */
 export function getEmbeddingColorPalette(
-  sdkColors?: MetabaseColors,
+  sdkColors: MetabaseColors = {},
```

Comment:
This fixes the issue where passing an empty colors `colors: {}` results in wrong colors.

If you override `brand` color on the instance and pass `colors: {}`, previously, the brand color would turn blue instead of your set `brand` color.